.github/workflows/ballerina-central-stats.yml

```diff
@@ -0,0 +1,30 @@
+name: Dashboard Extractor
+run-name: Testing out GitHub Actions Now
+on: [push]
+# on:
+#   workflow_dispatch:
+#   schedule:
+#     - cron: '30 18 * * *'   # 00:00 in LK time (GMT+5:30)
+
+
+jobs:
+  nballerina-build:
```

Comment:
change names to valid ones :D

Reply:
Fixed :) https://github.com/ballerina-platform/ballerina-release/pull/2576/commits/dfc15dbb74e88ba2c5a0393162b2316a9ce38169